docker/Dockerfile.aarch64-linux-android

```diff
@@ -10,27 +10,51 @@ RUN /cmake.sh
 COPY xargo.sh /
 RUN /xargo.sh
 
+COPY qemu.sh /
+RUN /qemu.sh aarch64
+
+ARG ANDROID_NDK=r21d
```

Comment:
with https://github.com/rust-lang/rust/issues/103673 / https://github.com/rust-lang/rust/pull/102332

we should probably target 25, however it seems it may be reverted and incrementally be deployed, beginning with 21